projects/canopy/src/lib/button/button.component.scss

```diff
@@ -40,10 +40,7 @@
 
   &__spinner {
     position: absolute;
-    left: 0;
-    right: 0;
-    top: 0;
-    bottom: 0;
+    inset: 0;
```

Comment:
I think this SCSS change could be in it's own commit, as it's not part of the unit tests?

Reply:
Sure, I will tidy up the commits and make them more granular. Thanks! :)